test/fixture/snapshots/test-sourcemaps/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+	"build": "tsc"
```

Comment:
Yes it does, I don't think there will be any need to run build here for the tests anyway. So I fixed the indentation and removed "scripts".